src/pages/storage/forms/DuplicateVolumeForm.tsx

```diff
@@ -102,6 +106,7 @@ const DuplicateVolumeForm: FC<Props> = ({ volume, close }) => {
       project: volume.project,
       copySnapshots: true,
       pool: volume.pool,
+      location: "",
```

Comment:
```suggestion
      location: isRemoteStorage(pool?.driver ?? "") ? "" : volume.location,
```
We need to initialize this for cluster member local volumes. Otherwise the cluster member selector will show the wrong default value on opening the duplication modal.

Reply:
I don't think this is necessary as the selector already only renders when the Pool driver is not remote storage, so rendering a conditional based on this will never be visible. Am I missing something else?

Reply:
The issue is with a custom volume on a non-remote pool. So the cluster member selector will be visible and show the first cluster member alphabetically. But the volume could live on the 2nd or 3rd or any other cluster member. When submitting the form without changing the member, the member of the volume will be used. So the selected member when launching the modal must match the volume node.

Reply:
https://github.com/user-attachments/assets/4018c915-d709-4b5c-9ce9-c548875657d3

Reply:
In fact, most cases having anything other than "" in this field results in the following error when attempting to select the default value:

![image](https://github.com/user-attachments/assets/82caffae-5be5-4583-a922-25d94671a2a6)

Whilst the code suggestion works for other values, it does not work when selecting the default value. I believe this has something to do with the case where the source location should only be set if the target member is not the same as the source, otherwise conflicts arise. Therefore, having a starting value of "" eliminates this.

Please advise how you would like me to proceed.

Reply:
The suggestions above are correct. The issue is with the lines below, simplify them to this to make it succeed:
```
      const targetProject = values.project;
      const targetMember = values.location;
```

Reply:
This works! Thank you for the correction.